stories/atoms/components/babylonEngine/space.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable @typescript-eslint/prefer-readonly-parameter-types */
+import type { Behavior, Engine, Mesh, Nullable, Observer } from '@babylonjs/core'
+import {
+  ArcRotateCamera,
+  Color3,
+  CubeTexture,
+  HemisphericLight,
+  MeshBuilder,
+  Scene,
+  StandardMaterial,
+  Texture,
+  Vector3
+} from '@babylonjs/core'
+import type { EngineCreatedCallback, RenderCallback } from 'ui/atoms/babylonEngine/BabylonEngine'
+import marsTexture from './assets/mars_1k_color.jpg'
```

Comment:
`Stories` folder already has an `asset` folder at this root, maybe we could move `babylonEngine` assets into it? What do you think?

Reply:
It depends. I like that the resources (mdx, assets, scripts) related to a story are all in the same folder, mainly because a good practice is to organize by business domaine rather than by technical domain. Of course if the resources are shared between several stories, it is different.

Reply:
I see your point and I agree with it. Maybe we could later remove the root asset folder to follow that architecture.